system_metrics_collector/CMakeLists.txt

```diff
@@ -61,37 +64,30 @@ if(BUILD_TESTING)
   ament_target_dependencies(test_collector rclcpp)
 
   ament_add_gtest(test_linux_cpu_measurement_node
-          test/system_metrics_collector/test_linux_cpu_measurement.cpp)
+    test/system_metrics_collector/test_linux_cpu_measurement.cpp)
+  target_include_directories(test_linux_cpu_measurement_node PRIVATE
+    ${metrics_statistics_msgs_INCLUDE_DIRS})
   target_link_libraries(test_linux_cpu_measurement_node system_metrics_collector)
   ament_target_dependencies(test_linux_cpu_measurement_node rclcpp)
 
   ament_add_gtest(test_linux_memory_measurement_node
-          test/system_metrics_collector/test_linux_memory_measurement.cpp)
+    test/system_metrics_collector/test_linux_memory_measurement.cpp)
+  target_include_directories(test_linux_memory_measurement_node PRIVATE
+    ${metrics_statistics_msgs_INCLUDE_DIRS})
   target_link_libraries(test_linux_memory_measurement_node system_metrics_collector)
   ament_target_dependencies(test_linux_memory_measurement_node rclcpp)
 
   ament_add_gtest(test_moving_average_statistics
-          test/moving_average_statistics/test_moving_average_statistics.cpp)
+    test/moving_average_statistics/test_moving_average_statistics.cpp)
   target_link_libraries(test_moving_average_statistics system_metrics_collector)
   ament_target_dependencies(test_moving_average_statistics rclcpp)
 
   ament_add_gtest(test_periodic_measurement_node
     test/system_metrics_collector/test_periodic_measurement_node.cpp)
+  target_include_directories(test_periodic_measurement_node PRIVATE
+    ${metrics_statistics_msgs_INCLUDE_DIRS})
   target_link_libraries(test_periodic_measurement_node system_metrics_collector)
   ament_target_dependencies(test_periodic_measurement_node rclcpp)
-
-  install(TARGETS
-    test_moving_average_statistics
-    DESTINATION
-    lib/${PROJECT_NAME})
-  install(TARGETS
-    test_collector
-    DESTINATION
-    lib/${PROJECT_NAME})
-  install(TARGETS
-    test_periodic_measurement_node
-    DESTINATION
-    lib/${PROJECT_NAME})
```

Comment:
Why are you removing this? Also, this seems unrelated.

Reply:
This change is removed from this PR.